11-checkout.md

```diff
@@ -1,119 +0,0 @@
----
-layout: lesson
-subtitle: Instructor Checkout Procedure
----
-> ## Motivating Question
-> 
-> What are the final steps in becoming an instructor trainer?
-{: .question}
-
-The final steps in getting certified are as follows:
-
-1. Pick one of the lessons we are currently accepting trainee requests on
-   and submit a *small* change.
-   (Your trainer will provide you with a list of lessons.)
-   If you want to teach Software Carpentry,
-   you must submit the change as a pull request on GitHub
-   and send the link to your trainer.
```

Comment:
Sorry to be a pain, I'd still say that we should have some general email such as training@... and trainees should cc this email in. If their trainer is not available (could be for a long while), they are stuck with their checkout.